packages/paste-website/src/components/site-wrapper/site-header/SiteHeaderLogo.tsx

```diff
@@ -38,39 +55,38 @@ const SiteHeaderLogo: React.FC<SiteHeaderLogoProps> = ({title, subtitle}) => {
       borderRightWidth={['borderWidth0', 'borderWidth0', 'borderWidth10']}
       minWidth={subtitle ? 'sizeSidebar' : 'size0'}
     >
-      <MediaObject verticalAlign="center">
-        <MediaFigure spacing="space40">
-          {/* <PasteIconInverse color={theme.textColors.colorTextInverse} display="block" size={42} /> */}
-          <PasteIconPride display="block" size={42} />
-        </MediaFigure>
-        <MediaBody>
-          <Text as="div" fontSize="fontSize40" lineHeight="lineHeight40" color="colorTextInverse">
-            <StyledLink
-              to="/"
-              onClick={() =>
-                trackCustomEvent({
-                  category: 'Top Navigation',
-                  action: 'click-paste-logo',
-                  label: 'Paste logo',
-                })
-              }
-            >
+      <StyledLink
+        href="/"
+        onClick={() =>
+          trackCustomEvent({
+            category: 'Top Navigation',
+            action: 'click-paste-logo',
+            label: 'Paste logo',
+          })
+        }
+        onMouseEnter={() => setPasteIcon(pasteRedIcon)}
+        onMouseLeave={() => setPasteIcon(pastePrideIcon)}
```

Comment:
If you didn't want to mess around with state you could probably use `backgroundImage="url()"` and on `_hover={{backgroundImage: 'url()'}}`.

Reply:
I'm not sure how Gatsby would bundle that so I could be wrong, but the css hover approach may cause a flicker on hover as the image is loaded for the first time unless a sprite is used.

Reply:
Since we wanted to transition between the two states, I left this approach, with some modifications.